src/Components/Publishing/RelatedArticles/RelatedArticlesCanvas.tsx

```diff
@@ -77,7 +87,17 @@ const ArticlesWrapper = styled.div`
     &:last-child {
       margin-right: 0;
     }
+    ${pMedia.xl`
+      margin: 0 10px;
+      &:first-child {
+        margin-left: 20px;
+      }
+      &:last-child {
+        border-right: 20px solid white;
+      }
+    `}
   }
+  ${props => props.isMobile && "-webkit-overflow-scrolling: touch;"}
```

Comment:
`styled-components` _should_ be able to auto-prefix this but I've also seen [issues](https://github.com/artsy/reaction/pull/186#pullrequestreview-47033980) with it in the past. Curious if this one is also the case.

Reply:
Looks like this property only exists in webkit (is for ios scrolling), so I believe this is the only way to declare.